c/authService.c

```diff
@@ -66,6 +67,19 @@
 
 static int serveAuthCheck(HttpService *service, HttpResponse *response);
 
+const char* getProfileNameFromRequest(char *url, const char *method, int instanceID);
+
+const char* makeProfileName(
+  char *type,
```

Comment:
If you don't change the memory behind these pointer, make them `const` to better show the intentions.